mods/ts/rules/defaults.yaml

```diff
@@ -497,6 +497,7 @@
 			TopLeft: -512, -512
 			BottomRight: 512, 512
 			VerticalTopOffset: 768
+	AppearsOnMapPreview:
```

Comment:
The various bridge defaults, rocks, and the veinhole monster appear to need this too - i'm finding all these cases by assuming anything with `RadarColorFromTerrain` also needs to specify `AppearsOnMapPreview` with the same terrain type.